cmd/npmExecuteScripts.go

```diff
@@ -88,13 +88,12 @@ func runNpmExecuteScripts(utils npmExecuteScriptsUtilsInterface, options *npmExe
 			return err
 		}
 
-		// set in each directory to respect existing config in rc files
-		err = npm.SetNpmRegistries(
-			&npm.RegistryOptions{
-				DefaultNpmRegistry: options.DefaultNpmRegistry,
-				SapNpmRegistry:     options.SapNpmRegistry,
-			}, execRunner)
-
+		npmExecutorOptions := npm.ExecutorOptions{DefaultNpmRegistry: options.DefaultNpmRegistry, SapNpmRegistry: options.SapNpmRegistry, ExecRunner: execRunner}
+		npmExecutor, err := npm.NewExecutor(npmExecutorOptions)
+		if npmExecutor == nil {
+			log.Entry().Errorln("Can't get npm executor")
+		}
```

Comment:
Error is ignored... but should be removed anyway. The nil-check should also be removed.